src/instructlab/sdg/generate_data.py

```diff
@@ -291,16 +347,18 @@ def generate_data(
     if generated_data is None:
         generated_data = []
 
-    _gen_train_data(logger, generated_data, os.path.join(output_dir, output_file_train))
+    _gen_train_data(
+        logger,
+        generated_data,
+        os.path.join(output_dir, output_file_train),
+        os.path.join(output_dir, output_file_messages_train),
+    )
 
     # TODO
     # This is for backwards compatibility. The file existing previously, so we'll keep it for now.
     # I believe the github bot assumes it is present for presenting generated data to a taxonomy
     # reviewer or contributor. Otherwise, I don't see a consumer of it in this repo or the
     # `ilab` CLI.
-    _gen_train_data(
-        logger, generated_data, os.path.join(output_dir, output_file_generated)
-    )
```

Comment:
Did you mean to remove this?

Reply:
yes, removed that. thanks!

Reply:
I had introduced a duplicate _gen_train_data function call, removed that. I am not 100% sure what the comment implies but from what I understand we still want to keep it for backwards compatibility the same reasons we have the simple ilab generate? now only additionally it will also generate the messages file required for the full pipeline

Reply:
As of this change `output_file_generated` is no longer created

Reply:
oh, I understand the intended functionality. Let me fix and test this